## `nixpkgs-review` result for [#430394](https://github.com/NixOS/nixpkgs/pull/430394)

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 430394 `

Logs: https://github.com/mdaniels5757/nixpkgs-review-gha/actions/runs/16694711365


---
### `x86_64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>matrix-authentication-service</li>
    <li>open-policy-agent</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>open-policy-agent</summary>
<pre>ok  	github.com/open-policy-agent/opa/v1/storage/internal/errors	0.004s [no tests to run]
?   	github.com/open-policy-agent/opa/v1/storage/internal/ptr	[no test files]
ok  	github.com/open-policy-agent/opa/v1/test/authz	0.049s
?   	github.com/open-policy-agent/opa/v1/test/cases	[no test files]
?   	github.com/open-policy-agent/opa/v1/test/cases/internal/fmtcases	[no test files]
?   	github.com/open-policy-agent/opa/v1/test/cases/internal/keywordrefs	[no test files]
ok  	github.com/open-policy-agent/opa/v1/test/scheduler	0.030s
?   	github.com/open-policy-agent/opa/v1/test/wasm/cmd/wasm-rego-testgen	[no test files]
ok  	github.com/open-policy-agent/opa/v1/tester	98.813s
--- FAIL: TestGraphQLParseSchemaAlloc (0.00s)
    --- FAIL: TestGraphQLParseSchemaAlloc/default_schema (0.00s)
        graphql_test.go:906: Parsing schema 'default schema' expected alloc < 1048576, got 18446744073708453264
2025/08/02 14:57:04 http: TLS handshake error from 127.0.0.1:37130: remote error: tls: bad certificate
2025/08/02 14:57:04 http: TLS handshake error from 127.0.0.1:37140: remote error: tls: bad certificate
2025/08/02 14:57:04 http: TLS handshake error from 127.0.0.1:37142: remote error: tls: bad certificate
2025/08/02 14:57:04 http: TLS handshake error from 127.0.0.1:37152: remote error: tls: bad certificate
2025/08/02 14:57:04 http: TLS handshake error from 127.0.0.1:56710: remote error: tls: bad certificate
FAIL
FAIL	github.com/open-policy-agent/opa/v1/topdown	4.952s
FAIL</pre>
</details>
</details>


---
### `aarch64-linux`
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>matrix-authentication-service</li>
    <li>open-policy-agent</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>open-policy-agent</summary>
<pre>ok  	github.com/open-policy-agent/opa/v1/storage/internal/errors	0.003s [no tests to run]
?   	github.com/open-policy-agent/opa/v1/storage/internal/ptr	[no test files]
ok  	github.com/open-policy-agent/opa/v1/test/authz	0.046s
?   	github.com/open-policy-agent/opa/v1/test/cases	[no test files]
?   	github.com/open-policy-agent/opa/v1/test/cases/internal/fmtcases	[no test files]
?   	github.com/open-policy-agent/opa/v1/test/cases/internal/keywordrefs	[no test files]
ok  	github.com/open-policy-agent/opa/v1/test/scheduler	0.029s
?   	github.com/open-policy-agent/opa/v1/test/wasm/cmd/wasm-rego-testgen	[no test files]
ok  	github.com/open-policy-agent/opa/v1/tester	126.906s
--- FAIL: TestGraphQLParseSchemaAlloc (0.00s)
    --- FAIL: TestGraphQLParseSchemaAlloc/default_schema (0.00s)
        graphql_test.go:906: Parsing schema 'default schema' expected alloc < 1048576, got 18446744073707329472
2025/08/02 14:57:11 http: TLS handshake error from 127.0.0.1:52542: remote error: tls: bad certificate
2025/08/02 14:57:11 http: TLS handshake error from 127.0.0.1:52556: remote error: tls: bad certificate
2025/08/02 14:57:11 http: TLS handshake error from 127.0.0.1:52572: remote error: tls: bad certificate
2025/08/02 14:57:11 http: TLS handshake error from 127.0.0.1:52578: remote error: tls: bad certificate
2025/08/02 14:57:11 http: TLS handshake error from 127.0.0.1:43946: remote error: tls: bad certificate
FAIL
FAIL	github.com/open-policy-agent/opa/v1/topdown	3.437s
FAIL</pre>
</details>
</details>


---
### `x86_64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>matrix-authentication-service</li>
    <li>open-policy-agent</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-darwin`</summary>
<details>
<summary>open-policy-agent</summary>
<pre>{"client_addr":"[::1]:49284","level":"info","msg":"Sent response.","req_id":2,"req_method":"PUT","req_path":"/v1/policies/mod1","resp_bytes":861,"resp_duration":0.648875,"resp_status":400,"time":"2025-08-02T14:53:03Z"}
{"level":"info","msg":"Shutting down...","time":"2025-08-02T14:53:03Z"}
{"level":"info","msg":"Server shutdown.","time":"2025-08-02T14:53:03Z"}
{"addrs":["localhost:8181"],"diagnostic-addrs":[],"level":"info","msg":"Initializing server.","time":"2025-08-02T14:53:03Z"}
{"client_addr":"127.0.0.1:49285","level":"info","msg":"Received request.","req_id":1,"req_method":"GET","req_path":"/health","time":"2025-08-02T14:53:03Z"}
{"client_addr":"127.0.0.1:49285","level":"info","msg":"Sent response.","req_id":1,"req_method":"GET","req_path":"/health","resp_bytes":3,"resp_duration":0.36875,"resp_status":200,"time":"2025-08-02T14:53:03Z"}
{"level":"info","msg":"Test server ready and listening on: http://127.0.0.1:8181","time":"2025-08-02T14:53:03Z"}
{"client_addr":"127.0.0.1:49286","level":"info","msg":"Received request.","req_id":2,"req_method":"PUT","req_path":"/v1/policies/mod2","time":"2025-08-02T14:53:03Z"}
{"client_addr":"127.0.0.1:49286","level":"info","msg":"Sent response.","req_id":2,"req_method":"PUT","req_path":"/v1/policies/mod2","resp_bytes":711,"resp_duration":0.254583,"resp_status":400,"time":"2025-08-02T14:53:03Z"}
{"level":"info","msg":"Shutting down...","time":"2025-08-02T14:53:03Z"}
{"level":"info","msg":"Server shutdown.","time":"2025-08-02T14:53:03Z"}
{"addrs":["localhost:8181"],"diagnostic-addrs":[],"level":"info","msg":"Initializing server.","time":"2025-08-02T14:53:03Z"}
{"client_addr":"127.0.0.1:49287","level":"info","msg":"Received request.","req_id":1,"req_method":"GET","req_path":"/health","time":"2025-08-02T14:53:03Z"}
{"client_addr":"127.0.0.1:49287","level":"info","msg":"Sent response.","req_id":1,"req_method":"GET","req_path":"/health","resp_bytes":3,"resp_duration":0.308125,"resp_status":200,"time":"2025-08-02T14:53:03Z"}
{"level":"info","msg":"Test server ready and listening on: http://127.0.0.1:8181","time":"2025-08-02T14:53:03Z"}
--- FAIL: TestRunServerUploadPolicy (0.44s)
    --- FAIL: TestRunServerUploadPolicy/v1,_v1_policy (0.11s)
        run_test.go:320: Unexpected error: Failed to PUT the test policy: Put "http://127.0.0.1:8181/v1/policies/mod3": dial tcp 127.0.0.1:8181: connect: operation not permitted
FAIL	github.com/open-policy-agent/opa/cmd	4.151s
FAIL</pre>
</details>
</details>


---
### `aarch64-darwin` (sandbox = true)
<details>
  <summary>:x: 2 packages failed to build:</summary>
  <ul>
    <li>matrix-authentication-service</li>
    <li>open-policy-agent</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-darwin`</summary>
<details>
<summary>open-policy-agent</summary>
<pre>ok  	github.com/open-policy-agent/opa/v1/plugins/rest	4.756s
ok  	github.com/open-policy-agent/opa/v1/plugins/server/decoding	0.008s
ok  	github.com/open-policy-agent/opa/v1/plugins/server/encoding	0.009s
ok  	github.com/open-policy-agent/opa/v1/plugins/server/metrics	0.009s
ok  	github.com/open-policy-agent/opa/v1/plugins/status	1.053s
ok  	github.com/open-policy-agent/opa/v1/profiler	0.142s
ok  	github.com/open-policy-agent/opa/v1/refactor	0.018s
ok  	github.com/open-policy-agent/opa/v1/rego	1.250s
ok  	github.com/open-policy-agent/opa/v1/repl	0.109s
?   	github.com/open-policy-agent/opa/v1/resolver	[no test files]
ok  	github.com/open-policy-agent/opa/v1/runtime	0.963s
ok  	github.com/open-policy-agent/opa/v1/schemas	0.008s
ok  	github.com/open-policy-agent/opa/v1/sdk	0.377s
?   	github.com/open-policy-agent/opa/v1/sdk/test	[no test files]
# github.com/open-policy-agent/opa/v1/server [github.com/open-policy-agent/opa/v1/server.test]
v1/server/server_bench_test.go:26:10: undefined: newReqV1
v1/server/server_bench_test.go:35:50: undefined: fixture
v1/server/server_bench_test.go:68:10: undefined: fixture
FAIL	github.com/open-policy-agent/opa/v1/server [build failed]
FAIL</pre>
</details>
</details>

--- a/pkgs/by-name/op/open-policy-agent/package.nix
+++ b/pkgs/by-name/op/open-policy-agent/package.nix
@@ -53,6 +53,9 @@ buildGoModule (finalAttrs: {
         # This test depends on the metrics available in go not changing. This is a bit
         # too unstable for us updating go independently.
         "TestJSONSerialization"
+
+        # Flaky
+        "TestGraphQLParseSchemaAlloc"
       ]
       ++ lib.optionals stdenv.hostPlatform.isDarwin [
         # Skip tests that require network, not available in the darwin sandbox

